app/hotels/src/HotelsFilterContext.js

```diff
@@ -98,9 +99,16 @@ class Provider extends React.Component<Props, State> {
     }, this.setActiveFilters);
   };
 
-  render = () => (
-    <ContextProvider value={this.state}>{this.props.children}</ContextProvider>
-  );
+  render() {
+    return (
+      <ContextProvider value={this.state}>
+        {this.props.children}
+      </ContextProvider>
+    );
+  }
 }
 
+export const withHotelsFilterContext = (select: State => Object) =>
```

Comment:
Would it be possible to say that `select` returns an object with keys a subset of the keys of `State`?

Reply:
Well, I guess you might want to rename sometimes...

Reply:
it does not have to be a subset of state e.g.

```
const select = ({ currency, checkin, checkout, currentSearchStats }) => ({
  currency,
  checkin,
  checkout,
  currentSearchStats,
  daysOfStay: calculateDaysOfStay(checkin, checkout),
});
```